generators/chipyard/src/main/scala/upf/UPFGen.scala

```diff
@@ -0,0 +1,263 @@
+package chipyard.upf
+
+import freechips.rocketchip.diplomacy.LazyModule
+
+import java.io.FileWriter
+import java.nio.file.{Paths, Files}
+import scala.collection.mutable.ListBuffer
+import scalax.collection.mutable.Graph
+import scalax.collection.GraphPredef._, scalax.collection.GraphEdge._
```

Comment:
Our general convention is to put the Scala/Java imports before the Rocket Chip (Chipyard, etc) imports.